tests/conftest.py

```diff
@@ -905,15 +905,15 @@ def is_port_in_use(host: str, port: int) -> bool:
 def _make_libp2p_connection(
     data_dir: str = TEMP_LIBP2P_TEST_DIR,
     port: Optional[int] = None,
-    host: Optional[str] = DEFAULT_HOST,
+    host: str = DEFAULT_HOST,
     relay: bool = True,
     delegate: bool = False,
     mailbox: bool = False,
     entry_peers: Optional[Sequence[MultiAddr]] = None,
     delegate_port: Optional[int] = None,
-    delegate_host: Optional[str] = DEFAULT_HOST,
+    delegate_host: str = DEFAULT_HOST,
     mailbox_port: Optional[int] = None,
-    mailbox_host: Optional[str] = DEFAULT_HOST,
+    mailbox_host: str = DEFAULT_HOST,
```

Comment:
removed the `Optional` based on [feedback in #230](https://github.com/valory-xyz/open-aea/pull/230#discussion_r926450743)